### PR TITLE
addiing cpp and header files of my elections project + small descrip

### PR DESCRIPTION
**Pull request by woodooi**
my 5 lab, thanks god

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#include <iostream>
+#include <C:\Users\Master\source\repos\Candidates\Candidates\classCandidates.h>
```
Comment (ViktorYasin) [inline main.cpp]: The link was incorrectly reset

File: classCandidates.h [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+};
+class Elections
+{
+public:
+	void sorted(Candidate* candidates)
+	{
+		bubblesort(candidates);
+		outWinner(candidates);
+	}
+	void bubblesort(Candidate candidates[])
```
Comment (ViktorYasin) [inline classCandidates.h]: Incorrect function name

**Review by ViktorYasin — CHANGES_REQUESTED** [review]

**Review by ViktorYasin — APPROVED** [review]

File: classCandidates.h [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+};
+class Elections
+{
+public:
+	void sorted(Candidate* candidates)
+	{
+		bubblesort(candidates);
+		outWinner(candidates);
+	}
+	void bubble_sort(Candidate candidates[])
```
Comment (zveres-ss) [inline classCandidates.h:53]: bubble_sort
but
outWinner
getVotes

which style is final?

File: classCandidates.h [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+	{
+		cout <<"The winner of the election is " << name <<" with "<<votesNum<<" votes" << endl;
+	}
+};
+class Elections
+{
+public:
+	void sorted(Candidate* candidates)
+	{
+		bubblesort(candidates);
```
Comment (zveres-ss) [inline classCandidates.h:50]: you changed method name to bubble_sort, code will not compile

**Review by zveres-ss — CHANGES_REQUESTED** [review]

**Pull request merged**